__tests__/app.test.js

```diff
@@ -41,3 +41,41 @@ describe("GET api/categories", () => {
       });
   });
 });
+
+describe("GET api/reviews/review_id", () => {
+  test("Status 200 - responds with review object with correct properties", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then(({ body: { review } }) => {
+        expect(review).toBeInstanceOf(Object);
```

Comment:
do we need this assertion if you're also testing it's an object immediately below?